src/lib.rs

```diff
@@ -124,6 +124,10 @@
 //! maximum redirect chain of 10 hops. To customize this behavior, a
 //! [`redirect::Policy`][redirect] can be used with a `ClientBuilder`.
 //!
+//! **NOTE**: unlike some clients like `libcurl`, when `reqwest`
```

Comment:
It looks like the [default in curl](https://curl.se/libcurl/c/CURLOPT_UNRESTRICTED_AUTH.html) is also to restrict sending auth headers only to the original host. Python's Requests also [removes auth headers](https://docs.python-requests.org/en/latest/user/quickstart/#custom-headers).

Reply:
Shoot, you're right! I didn't realize that I had a `-L` in my curl command and that `requests` was reading from `~/.netrc`. I'm sorry for wasting your time!